pkg/aspect/accessLogsManager.go

```diff
@@ -117,14 +118,29 @@ func (e *accessLogsWrapper) Close() error {
 }
 
 func (e *accessLogsWrapper) Execute(attrs attribute.Bag, mapper expr.Evaluator, ma APIMethodArgs) Output {
-	labels, err := evalAll(e.labels, attrs, mapper)
-	if err != nil {
-		return Output{Status: status.WithError(fmt.Errorf("failed to eval labels for log %s with err: %s", e.name, err))}
+
+	labels := make(map[string]interface{}, len(e.labels))
+	for name, exp := range e.labels {
+		v, err := mapper.Eval(exp, attrs)
+		if err == nil {
+			labels[name] = v
+			continue
+		}
+		if name == "timestamp" {
```

Comment:
Is there a way we can accomplish the same thing without relying on an explicit name for the label? e.g. `if expr.TypeOf(exp) == time {... }`? I suppose we don't have the correct interfaces to do that just right now (after I add descriptors I plan to expose types for expressions on the Evaluator interface). In theory the user could call this whatever they want.